pkg/kms/service.go

```diff
@@ -179,6 +184,63 @@ func (s *service) VerifyMAC(keystoreID, keyID string, mac, data []byte) error {
 	return nil
 }
 
+// WrapKey wraps cek for the recipient with public key 'recipientPubKey'.
+func (s *service) WrapKey(keystoreID, keyID string, cek, apu, apv []byte,
+	recipientPubKey *crypto.PublicKey) (*crypto.RecipientWrappedKey, error) {
+	if keyID != "" {
+		kh, err := s.getKeyHandle(keystoreID, keyID)
+		if err != nil {
+			return nil, err
```

Comment:
Looks like there are some unwrapped errors in this file

Reply:
I am trying to wrap only once when code passes package boundary - in this case getKeyHandle already does the wrapping.